joomla/joomla-cms
# Add mobile breakpoint config to Cassiopeia template and fix padding issues

Pull Request for Issue # .

### Summary of Changes
This PR addresses two issues:

The Cassiopeia template has a left and a right position. At the moment the breakpoint at which the left and right position will stack is way to small and not functional. This is the same with modules in top-a, top-b, bottom-a and bottom-b. With this PR I made the mobile breakpoint configurable in the template settings.

![template-parameters](https://user-images.githubusercontent.com/5610413/124272532-2976b480-db3f-11eb-850c-da351089da84.jpg)

Modules is top-a, top-b, bottom-a and bottom-b don't align with the sidebar-left and sidebar-right position of the Cassiopeia template. This PR fixes this.

![align-modules](https://user-images.githubusercontent.com/5610413/124272549-2ed3ff00-db3f-11eb-97c0-3195f75ad609.jpg)

The extra mobile classes in the body, adds also a lot of extra flexibility in styling a custom template.

### Testing Instructions
1. Apply the patch and compile the CSS
2. Add multiple modules to the sidebar-left, sidebar-right, top-a, top-b, bottom-a and bottom-b
3. See if the modules in top-a, top-b, bottom-a and bottom-b align with the  sidebar-left and sidebar-right
4. Set the mobile breakpoint in te template an see if the modules respond to the set breakpoint

### Actual result BEFORE applying this Pull Request
See description above

### Expected result AFTER applying this Pull Request
See description above

### Documentation Changes Required
No
